testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/LandingPage.java

```diff
@@ -0,0 +1,50 @@
+package org.keycloak.testsuite.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Backup Code Authentication test
+ *
+ * @author <a href="mailto:vnukala@redhat.com">Venkata Nukala</a>
+ */
+public class LandingPage extends LanguageComboboxAwarePage {
```

Comment:
This class is used in RecoveryAuthnCodesAuthenticatorTest, but this test does not use it for anything. So since this class is effectively not used anywhere, it can be possible removed from this PR?